install_files/ansible-base/group_vars/securedrop_monitor_server.yml

```diff
@@ -34,7 +34,7 @@ authd_iprules:
 
 # Declare Monitor Server as OSSEC server role.
 ossec_is_server: yes
-authd_rules:
+authd_iprules:
```

Comment:
Ahh my bad, I should have been more specific in my comment: `authd_iprules` is defined above already in this file (redefining it like so in this commit will cause the playbook to bail out when it gets [here](https://github.com/freedomofpress/securedrop/blob/a8915e0740caefc4664a509b3f39997c97a090b4/install_files/ansible-base/roles/ossec/tasks/register.yml#L78)). Since these authd rules do not change, we can safely remove this second definition.

Reply:
ahahah im slow. rebased and removed.